src/mlpack/bindings/julia/CMakeLists.txt

```
@@ -7,6 +7,16 @@ macro (not_found_return message)
   return()
 endmacro ()
 
+macro (post_julia_setup)
+  # If we are building Julia bindings, we have to end the 'module' declaration in
+  # mlpack.jl
```

```suggestion
  # mlpack.jl.
```
Sorry for the pedantry. :smile: